x/emissions/abci.go

```diff
@@ -12,28 +13,47 @@ import (
 
 func BeginBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 
-	reservesFactor, bondFactor, durationFactor := keeper.GetBlockRewardComponents(ctx)
-	blockRewards := reservesFactor.Mul(bondFactor).Mul(durationFactor)
+	emissonPoolBalance := keeper.GetReservesFactor(ctx)
```

Comment:
Why GetReservesFactor return a balance? Is it the right method used?

Reply:
ReserversFactor , is the balance of the emission module account , "The amount of token in reserve" . I decided not to rename the function to keep changes minimal

Reply:
typo emisson => emission